src/main/resources/attackvectors/ErrorBasedSQLInjectionPayload.properties

```diff
@@ -0,0 +1,16 @@
+ERROR_BASED_SQL_INJECTION_PAYLOAD_LEVEL_1=Error based SQL injection is a technique where an attacker tries to insert a malicious query into the input fields and receives a SQL or database syntax error.<br/>\
+ How to exploit this level?<br/>\
+ Import this in textbox<br/>\
```

Comment:
Better term than import is `input` or `insert`. Please try grammarly for english related help.